src/main/ibm-telemetry.ts

```diff
@@ -109,6 +109,9 @@ export class IbmTelemetry {
           [CustomResourceAttributes.ANALYZED_PATH]: `${repository.host ?? ''}/${
             repository.owner ?? ''
           }/${repository.repository ?? ''}`,
+          [CustomResourceAttributes.ANALYZED_OWNER_PATH]: `${repository.host ?? ''}/${
+            repository.owner ?? ''
+          }`,
```

Comment:
This template literal string looks weird with the newlines... but I see this is consistent with above and just Prettier formatting. At first I thought the string would have newlines. I now see it won't.